torch/optim/adadelta.py

```diff
@@ -105,13 +120,14 @@ def step(self, closure=None):
                 group["differentiable"],
             )
 
-            has_complex = self._init_group(group, params_with_grad, grads, square_avgs, acc_deltas)
+            has_complex = self._init_group(group, params_with_grad, grads, square_avgs, acc_deltas, state_steps)
 
             adadelta(
```

Comment:
missing capturable routing? Even if the function doesn't vary depending on the flag, it would be good to route it through for API consistency + for potential error handling